apps/omg_watcher_rpc/test/omg_watcher_rpc/web/validators/transaction_constraints_test.exs

```diff
@@ -0,0 +1,87 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherRPC.Web.Validator.TransactionConstraintsTest do
+  use ExUnit.Case, async: true
+
+  alias OMG.Eth.Encoding
+  alias OMG.Eth.RootChain
+  alias OMG.WatcherRPC.Web.Validator.TransactionConstraints
+
+  @eth RootChain.eth_pseudo_address()
+  @zero_metadata <<0::256>>
+
+  describe "parse/1" do
```

Comment:
Can you add test for parsing `tx_types` as well? Thank u

Reply:
Added